frontend/src/components/Configuration/DroneConfiguration.jsx

```diff
@@ -357,6 +361,18 @@ export default function DroneConfiguration (droneData)  {
                             </StyledSelect>
                         </Grid>
 
+                        {/* Drone Image Field */}
+                        <Grid item xs={12}>
+                            {selectedDroneType && (
+                                <Box mt={2}>
+                                    <img src={droneModels[selectedDroneType].find((m) => m.value === selectedModel)?.src}
```

Comment:
@Hoshi-Okami Bigger drone images occupies most of the screen space. SO reduce its dimensions, particularly height.  Also can we remove the white background for [Aurelia-X6-Pro.png](https://github.com/UAVLab-SLU/DRV_public/pull/46/files#diff-2da827bc7c406d4f4e9ba9fd1f42a328cd4a13dbe028f36b881f8bbb145eae50). It seems all other images have no background, and such images blended well into our background.

Reply:
![image-9](https://github.com/UAVLab-SLU/DRV_public/assets/173104244/9dc5455b-1bcb-43e6-95a2-1b931f729504)
![image-8](https://github.com/UAVLab-SLU/DRV_public/assets/173104244/3b0babbe-0a81-405e-9121-da6e0e9a7d26)